### PR TITLE
Re-adds the mapmerge instructions

### PR DESCRIPTION
**Pull request by Crazylemon64**
@tigercat2000 Removed these because they thought the tools were easy enough to use, I figured this won't hurt at all and would help newcomers make heads or tails of the system (#4178).

### CODE REVIEW

File: tools/mapmerge/install.txt [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+2. Make sure java is in your PATH. To test this, open git bash, and type "java". If it says unknown command, you need to add JAVA/bin to your PATH variable (A guide for this can be found at https://www.java.com/en/download/help/path.xml ).
+
+Committing
+1. Before starting to edit the map, double-click "prepare_map.bat" in the tools/mapmerge/ directory.
+2. After finishing your edit, and before your commit, double-click "clean_map.bat" in the tools/mapmerge/ directory.
+
+This will make sure in the new version of your map, no paths are needlessly changed, thus instead of 8000 lines changed you'll get 50 lines changed. This not only reduces size of your commit, it also makes it possible to get an overview of your map changes on the "files changed" page in your pull request.
+
+Merging
+The easiest way to do merging is to install the merge driver. For this, you can just double-click "install_driver.bat" in the tools/mapmerge/ directory.
```
Comment (tigercat2000) [inline tools/mapmerge/install.txt]: this entire section isn't possible with our current mapmerge

Comment (Crazylemon64) [inline tools/mapmerge/install.txt]: Ah, I'll rip out the merge driver bit then

**Comment by Iamgoofball** [comment]
>>>still using java map merge
>>>year of our lord 2016


upgrade to python babes, contact your local #coderbus represenative for more details

**Comment by monster860** [comment]
I would really love to have the mapmerge driver back. Once, I had to resolve a merge conflict, but a billion objects were removed. When I tried to tell DM to just remove them, it crashed. I ended up having to add the list of objects to the end of a code file so that DM wouldn't complain, and then I would be able to copy the shit over.

It fucking sucked.

**Comment by QQCucumber** [comment]
Instructions are good. Documentation is good.

**Pull request merged**